CONTRIBUTING.md

```diff
@@ -1,14 +1,49 @@
+# Contributing
+
+Welcome to our Github project! We are happy to have you here. We welcome contributions from anyone and everyone. By participating in this project, you agree to abide by our [Code of Conduct](./code_of_conduct.md).
+
+## Getting started
+
+For code-related instructions, please refer to the [README](./README.md) file. You can find issues in the project's issues page and you can submit pull requests to the project's pull requests page. If you are a first-time contributor, you may find value in filtering the issues that are labeled as "good first issue".
```

Comment:
We say "issues that are labeled as "good first issue".   Let's make sure we have a one or two of these in place son.

Reply:
Good idea. I have taken the idea of updating the packages so that we can use that as a good first issue. I have created the issue, but I haven't been able to tag it, see #1074